index.html

```diff
@@ -1,39 +1,60 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>Tom's Biography</title>
+    <title>About Tom</title>
     <link rel="stylesheet" type="text/css" href="style.css">
   </head>
   <body>
     <main>
       <section>
-        <h2>Experience</h2>
+        <h1>Biography</h1>
+        <h2>List of Experience</h2>
         <ul>
           <li>Zamboni Driver</li>
+          <li>Marketing Guy</li>
         </ul>
       </section>
       <section>
         <h2>Education</h2>
         <ul>
-          <li>Colorado State University</li>
+          <li>Fairview H.S.</li>
+          <li>Colorado State</li>
         </ul>
       </section>
       <section>
-      <h2>Goals</h2>
-        <ul>
-          <li>Graduate 201!</li>
-          <li>Graduate 301!</li>
-          <li>Graduate 401!</li>
-        </ul>
+        <h2>List of Goals</h2>
+        <ol>
+          <li>Code Fellows 201!</li>
+          <li>Code Fellows 301!</li>
+          <li>Code Fellows 401!</li>
+        </ol>
       </section>
       <section id ="guessing-game">
+        
         <p id = "user-Name"></p>
+
+
         <p id = "ans-one"></p>
+
+
         <p id = "ans-two"></p>
+
+
         <p id = "ans-three"></p>
+
+
         <p id = "ans-four"></p>
+
+
```

Comment:
Remove white space lines 33, 35, 36, 38, 39, 41, 42, 44, 45, 47, 48 (for convention). Also, close your spaces between equal signs and your id and src attributes (should be <p id="example">). For html, there shouldn't be any spaces (ok in Javascript).